openworld/sdk/generator/client/__main__.py

```diff
@@ -71,7 +71,7 @@ def generate_code(
     code_formatter = CodeFormatter(PythonVersion.PY_38, Path().resolve())
 
     template_vars: dict[str, object] = {"info": parser.parse_info()}
-    visitors: list[Visitor] = []
+    visitors: list[tuple[int, Visitor]] = []
```

Comment:
Why do we need to order the visitors now?

Reply:
Consider the following case:

- `Operations` visitor processes models, does some changes, for example changes `Include1` to `GetRegionInclude`.
- `Models` visitor does some processing and returns operations data, we need to ensure `Operations` visitor is executed before `Models` so we have `GetRegionInclude` instead of `Include1` before "static data" is returned.